src/pathfinding_service/cli.py

```diff
@@ -101,7 +101,11 @@ def main(  # pylint: disable-msg=too-many-arguments
             db_filename=state_db,
             matrix_servers=matrix_server,
         )
+        service.start()
+        log.debug("Waiting for service to start before accepting API requests")
+        assert service.startup_finished.wait(timeout=60), "PFS did not start within time."
```

Comment:
Is 60s enough?

Reply:
It took < 20s on my machine. Since most of it is latency, it should be similar on slower machines.